generate/generate_nircam.py

```diff
@@ -388,21 +388,11 @@
 
         create_jira_plots = True
         if create_jira_plots:
-            # # make figures for JWSTSIAF-160 Jira ticket
-            selected_aperture_names = [['NRCB5_GRISM256_F322W2','NRCB5_GRISM128_F322W2','NRCB5_GRISM64_F322W2','NRCB5_FULL',
-                                        'NRCA5_GRISM256_F322W2','NRCA5_GRISM128_F322W2','NRCA5_GRISM64_F322W2','NRCA5_FULL'],
-                                        ['NRCB2_GRISMTS256','NRCB2_GRISMTS128','NRCB2_GRISMTS64','NRCB2_FULL',
-                                         'NRCB4_GRISMTS256','NRCB4_GRISMTS128','NRCB4_GRISMTS64','NRCB4_FULL',
-                                         'NRCB5_FULL',
-                                         'NRCA1_GRISMTS256','NRCA1_GRISMTS128','NRCA1_GRISMTS64','NRCA1_FULL',
-                                         'NRCA3_GRISMTS256','NRCA3_GRISMTS128','NRCA3_GRISMTS64','NRCA3_FULL',
-                                         'NRCA5_FULL'],
-                                        ['NRCB2_FULL','NRCB2_GRISMTS256',
-                                         'NRCB4_GRISMTS256','NRCB4_FULL',
-                                         'NRCB5_GRISM256_F322W2','NRCB5_FULL',
-                                         'NRCA1_GRISMTS256','NRCA1_FULL',
-                                         'NRCA3_GRISMTS256','NRCA3_FULL',
-                                         'NRCA5_GRISM256_F322W2','NRCA5_GRISM256_F444W','NRCA5_FULL']
+            # # make figures for JWSTSIAF-206 Jira ticket
```

Comment:
@mgennaro when you want to plot new apertures, is it normal to remove prior apertures?

Reply:
Definitely. That part of the code would always change for any PR that updates apertures because the apertures we want to see/plot/tests will be specific to that PR. Nevertheless, that does not mean we need to merge those changes everytime, but that makes it easier to generate the plots specific for that PR.

No need to re-plot the old apertures every time (and/or to let the list of plotted apertures grow as we add more and more PRs).